docs/embed/gtm.md

```diff
@@ -128,13 +164,6 @@ Here's a list of events provided by VoteAmerica. Some events have variable param
   state: '[STATE]',
 }
 
-{
-  event: 'action-finish',
-  method: 'pdf',
-  tool: 'absentee',
-  state: '[STATE]',
-}
-
```

Comment:
Did this turn out to not be a real event? I'm curious why it got deleted.

Reply:
I should probably run this by you actually. 

But in the docs, this was listed twice, so I was really just removing a duplicate: 
```
{
  event: 'action-finish',
  method: 'pdf',
  tool: 'absentee',
  state: '[STATE]',
}
```

But I think the reason it was duplicated is that there are 2 JS files that produce this event - `GetMyForm.js` and `SignMyForm.js`. I was able to find and update the `GetMyForm` one, but I haven't figured out what triggers the other.